bootstore/src/schemes/v0/peer.rs

```diff
@@ -412,16 +413,17 @@ impl Node {
         &mut self,
         res: Result<(TcpStream, SocketAddr), std::io::Error>,
     ) {
+        let log = self.log.new(o!("stage" => SWITCH_ZONE_INIT_STAGE));
```

Comment:
This runs on all sleds, not just scrimlets, right? I'm not sure about marking a switch zone stage if that's the case.

Reply:
I'd be inclined to remove it, but I don't feel _super_ strongly about it. Maybe @smklein or @andrewjstone could take a glance and weigh in?

Reply:
I think naming bootstore p2p communication as "Switch zone initialization" is a bit misleading -- we use this protocol for trust quorum setup, which would happen (for one example) as we extend trust quorum to a new sled, totally unrelated to the switch zone. That could happen well after the system has been running, and it seems confusing to have all these logs labelled as "switch zone initialization".

I think I agree with @jgallagher here - adding this label to all bootstore communication seems like a bit of a mismatch.